analyzers/tests/SonarAnalyzer.Test/TestCases/ClassShouldNotBeEmpty.cs

```diff
@@ -126,4 +126,8 @@ class ImplementsMarker: IMarker { }          // Compliant - implements a marker
     struct EmptyStruct { }                       // Compliant - this rule only deals with classes
 
     enum EmptyEnum { }                           // Compliant - this rule only deals with classes
+
+    class SomeCommand { }                        // Compliant, ignored because of the suffix
+    class SomeEvent { }                          // Compliant, ignored because of the suffix
+    class SomeMessage { }                        // Compliant, ignored because of the suffix
```

Comment:
I suggest adding a few more test cases:
```cs
class SomeCommandAction { }        // Noncompliant
class SomeEventHandler { }              // Noncompliant
class SomeMessageReceived { }       // Noncompliant
```

Reply:
Should we check with `StringComparison.OrdinalIgnoreCase`? 
I would add at least 1 of the above UTs to test the `EndWith`, changing the implementation to `Contain` will not break any UTs as of now

Reply:
I don't think ignoring case is useful. I added UTs to document the behavior.